Added cloud9 instructions for helper-scripts lesson

cjhar opened the pull request:
## What does this PR do and why?

Added guidance to use Cloud9 dev environment for helper-scripts lesson

## Issue #, if available

## PR acceptance checklist

This checklist encourages us to confirm any changes have been analyzed to reduce risks in quality, performance, reliability, security, and maintainability.

- [ ] Added/Updated documentation if applicable.
- [X] Pre-commit checks passed.
- [X] Lint and Nag checks passed.
- [ ] If releasing a new version, have you bumped the version `make version part=<major|minor|patch>`?

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

cjhar commented:
Ready for review, thanks!

sdangol commented on content/basics/operations/helper-scripts/index.md:
```diff
@@ -304,11 +304,33 @@ In the example below, you will use `AvailabilityZone` property and parameter to
+  :::
+  1. Use the AWS CLI to create the stack. The required parameters `--stack-name` and `--template-body` have been pre-filled for you.
+  :::code{language=shell showLineNumbers=false showCopyAction=true}
+  aws cloudformation update-stack --stack-name cfn-workshop-ec2 \
```
```suggestion
  aws cloudformation update-stack --stack-name cfn-workshop-resources \
```

sdangol commented on content/basics/operations/helper-scripts/index.md:
```diff
@@ -304,11 +304,33 @@ In the example below, you will use `AvailabilityZone` property and parameter to
+  :::
+  1. If the `update-stack` command was successfully sent, CloudFormation will return `StackId`.
+  :::code{language=shell showLineNumbers=false showCopyAction=true}
+  "StackId": "arn:aws:cloudformation:us-east-1:123456789012:stack/cfn-workshop-ec2/96d87030-e809-11ed-a82c-0eb19aaeb30f"
```
```suggestion
  "StackId": "arn:aws:cloudformation:us-east-1:123456789012:stack/cfn-workshop-resources/96d87030-e809-11ed-a82c-0eb19aaeb30f"
```

sdangol commented on content/basics/operations/helper-scripts/index.md:
```diff
@@ -304,11 +304,33 @@ In the example below, you will use `AvailabilityZone` property and parameter to
+
+::::tab{id="local" label="Local development"}
 1. Open the **[AWS CloudFormation](https://console.aws.amazon.com/cloudformation)** link in a new tab and log in to your AWS account.
 1. Click on the stack name, for example **cfn-workshop-ec2**.
```
```suggestion
1. Click on the stack name, for example **cfn-workshop-resources**.
```

sdangol reviewed (changes requested):
Please update the naming conventions and change the command to create the stack

cjhar commented:
Thank you for review and suggestions, all incorporated and updated sample images. Note I removed the Availability Zone ID content as we don't need to trigger an initial EC2 instance replacement when we create all new resources. Estimated timing confirmed.

**Tested end2end**

sdangol commented on content/basics/operations/helper-scripts/index.md:
```diff
@@ -278,60 +278,52 @@ notify AWS CloudFormation when all the applications are installed and configured
-    ::alert[Notice that in **Change set preview**, the _Replacement_ condition of EC2 resource is **True**. Hence, the current EC2 instance will be terminated and replaced with a new one.]{type="info"}
-
-1. You can click the **refresh** button a few times until you see in the status **UPDATE_COMPLETE**.
+5. Verify the new instance was deployed and is functional
```
Please put inside the tabs

sdangol commented on content/basics/operations/helper-scripts/index.md:
```diff
@@ -381,10 +395,10 @@ You should see the AMI ID added to the page, similar to the picture below.
-1. In the **[CloudFormation console](https://console.aws.amazon.com/cloudformation)**, select the stack you have created in this lab. For example `cfn-workshop-ec2`.
+1. In the **[CloudFormation console](https://console.aws.amazon.com/cloudformation)**, select the stack you have created in this lab. For example `cfn-workshop-helper-scripts`.
 1. In the top right corner, click on **Delete**.
 1. In the pop-up window click on **Delete stack**.
```
```suggestion
1. In the pop-up window click on **Delete**.
```

sdangol reviewed (changes requested):
Added some more changes

sdangol reviewed (dismissed):
LGTM

rezabekf reviewed (approved):
LGMT
